### PR TITLE
Update plugins.md

### PR DESCRIPTION
**Pull request by Hamms**
Add https://github.com/code-dot-org/remark-redactable to the list of plugins

### CODE REVIEW

File: doc/plugins.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ See [Creating plugins][create] below.
     — highlight code blocks for **remark-react** with [lowlight](https://github.com/wooorm/lowlight)
 *   [`remark-redact`](https://github.com/seafoam6/remark-redact)
     — conceal text matching a regex
+*   [`remark-redactable`](https://github.com/code-dot-org/remark-redactable)
+    - write plugins to redact content from a markdown document, then restore it later
```
Comment (wooorm) [inline doc/plugins.md]:
```suggestion
    — write plugins to redact content from a markdown document, then restore it later
```

Use an em-dash here, now it’s another list item.

**Comment by Hamms** [comment]
Good catch! PTAL

File: doc/plugins.md [hunk 2]
```diff
@@ -139,6 +139,8 @@ See [Creating plugins][create] below.
     — highlight code blocks for **remark-react** with [lowlight](https://github.com/wooorm/lowlight)
 *   [`remark-redact`](https://github.com/seafoam6/remark-redact)
     — conceal text matching a regex
+*   [`remark-redactable`](https://github.com/code-dot-org/remark-redactable)
+    — write plugins to redact content from a markdown document, then restore it later
```
Comment (wooorm) [inline doc/plugins.md]:
```suggestion
    — write plugins to redact content from a markdown document,
    then restore it later
```

Oh no, now it complains about line length, how about this?

Comment (Hamms) [reply doc/plugins.md]: D'oh!

**Comment by wooorm** [comment]
Thank you! 🎉

**Comment by Hamms** [comment]
Thank _you_!

**Pull request merged**